xml/release-notes.xml

```diff
@@ -648,6 +648,70 @@ ServerArguments=-nolisten tcp -dpi <replaceable>DPI_SETTING</replaceable>
    </para>
   </sect2>
 
+  <sect2 xml:id="sec.desktop.qt-hidpi">
+   <!-- boo#1091103 -->
+   <title>Using Automatic Scaling in Qt Applications in Setups Which Mix High-DPI/Regular-DPI Monitors</title>
+   <para>
+    Qt supports automatic per-monitor scaling on X. It uses the DPI value of
+    the virtual X screen to calculate the font size for the primary monitor.
+    By default, this value is 96 DPI. It uses the relative DPI of the primary
+    monitor to derive font DPI for all other monitors.
+   </para>
+   <para>
+    Two widely used desktops will override this behavior of Qt, therefore this
+    note does not apply to them:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      GNOME will set DPI according to each monitor's reported scaling.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      KDE Plasma disables automatic scaling by default.
+     </para>
+    </listitem>
+   </itemizedlist>
+   <para>
+    On other desktops, this behavior of Qt can lead to undesirable situations
+    such as the following: If the primary display is High-DPI
+    (&gt;= 144 DPI), fonts in Qt applications that request scaling, such as
+    VLC, are effectively scaled to half the desired size on all monitors.
+    Applications which do not request scaling, such as &yast; (with default
+    settings), are not affected and use the same DPI value on all monitors.
+   </para>
+   <para>
+    You can use one of the following workarounds for this issue:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      Use a monitor with a regular DPI value as the primary monitor.
+      Applications that request scaling are then scaled appropriately on the
+      High-DPI monitor.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Set an appropriate font DPI (<literal>Xft.dpi</literal>) with the
+      configuration utility of your desktop or the following command:
+     </para>
+     <screen>echo Xft.dpi:<replaceable>DPI_VALUE</replaceable> | xrdb -nocpp -merge</screen>
+     <para>
+      Replace <replaceable>DPI_VALUE</replaceable> with an appropriate DPI
+      value for the primary monitor.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Disable automatic scaling:
+      <command>export QT_AUTO_SCREEN_SCALE_FACTOR=0</command>
```

Comment:
How is this helping? Won't we just have tiny fonts on the High-DPI monitor?

Reply:
Yes. It'll fix the appearance on the primary 96 DPI monitor though. The Xft.dpi workaround should cover most cases already, so you could probably remove this.